dataframe_image/_screenshot.py

```diff
@@ -112,26 +112,25 @@ def take_screenshot(self):
         with open(temp_html, "w", encoding="utf-8") as f:
             f.write(self.html)
 
-        with open(temp_img, "wb") as f:
```

Comment:
This line looks confusing in origin code, I think it should be removed.

Thanks a lot!